sci-biology/salmon/salmon-0.6.0.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
```

Comment:
Is cmake-utils now compatible with EAPI=6? Travis is still failing 2 days ago: https://github.com/gentoo-science/sci/pull/573